utils/luci-app-owm/files/owm.lua

```diff
@@ -53,6 +53,15 @@ function unlock()
 	os.execute("rm -f "..lockfile)
 end
 
+-- a location for the node is required to run
+if next(owm.get_position())==nil then
```

Comment:
I don't care about the extra flash space this takes. I care about readable code. `next(owm.get_position())==nil` is not quite intuitive.